test/runtests.jl

```diff
@@ -171,3 +171,5 @@ tests = [
         end
     end
 end;
+
+nothing;
```

Comment:
?

Reply:
otherwise the last result is printed in the REPL, and I find that disturbing. See also https://github.com/JuliaGraphs/Graphs.jl/pull/350/files#r1519487219